drivers/storage/portworx/testspec/autopilotDeployment.yaml

```diff
@@ -75,6 +82,15 @@ spec:
                     operator: Exists
                   - key: node-role.kubernetes.io/worker
                     operator: Exists
```

Comment:
Can a node be both `master` and a `worker`?
* is this intended for a single-node systems?

Reply:
Yes, it can be. IBM cloud deploys such a cluster with both master and worker labels and that is why we had to add this.